control/init.sh

```diff
@@ -67,8 +78,8 @@ function ansible-control-banner() {
     printf "${BLUE}update-code${NC} - update the ansible repositories (safely)\n"
     printf "${BLUE}workon ansible${NC} - activate the ansible virtual env\n"
     printf "${BLUE}commcare-cloud${NC} - CLI wrapper for ansible. See ${YELLOW}commcare-cloud -h${NC} for more details.\n"
-    printf "ap - Use ${YELLOW}commcare-cloud <env> ansible-playbook${NC} instead.\n"
-    printf "aps - Use ${YELLOW}commcare-cloud <env> ansible-playbook deploy_stack.yml${NC} instead.\n"
+    printf "ap - Alias for ${YELLOW}commcare-cloud <env> ansible-playbook${NC} e.g. ap production deploy_proxy.yml \n"
+    printf "aps - Alias for ${YELLOW}commcare-cloud <env> ansible-playbook deploy_stack.yml${NC} e.g. aps production \n"
```

Comment:
You've changed the expected arguments of these aliases, so they are not technically backward compatible with how they worked before.
```
# before
ENV=staging && aps --diff

# after
aps staging --diff
```
Seems like that could be confusing for anyone who's not paying close attention to all these changes recently.

Reply:
Personally I'd like to advocate for having the documented and recommended way to execute these commands be to use the `cchq` alias for `commcare-cloud` (@dannyroberts is working on documenting that now). `cchq` is pretty slick and actually requires less typing than before when paired with the new bash completion @dannyroberts implemented.

If you still want very terse aliases like `ap` and `aps`, what do you think about adding them to your bash profile?

Reply:
[`cchq` alias is now documented](https://github.com/dimagi/commcarehq-ansible/tree/master/commcare-cloud#optional-hook-up-the-bells-and-whistles)

Reply:
(I want to also give a shoutout to @javierwilson for doing the initial setup and most of the work for the commcare-cloud bash completion, which has proven totally awesome.)

Reply:
@calellowitz Worth weighing back in here? Good to decide whether we're going to go with this, close and go with @millerdev's suggestion, or something different.

Reply:
I would still like this included, as I do think it remains easier to use than the the new command (even with the slightly shorter alias) and I think encourages people to do the "right thing" (by making `deploy_stack.yml` the easiest possible command as it used to be). I am not committed to this version if you or @millerdev have an alternative suggestion, but I do think `deploy_stack` should be treated separately and made as easy as possible, and preferably different from other playbooks to encourage people to use it.

All that said, if there is strong resistance to that, I am happy to follow daniel's suggestion and add this to my own profile. I just think it might be useful/good for others.

Reply:
@dannyroberts forgot to mention